lib/evss/service.rb

```diff
@@ -9,12 +9,13 @@ class Service < Common::Client::Base
 
     def initialize(user)
       @user = user
+      @headers = headers_for_user(@user)
     end
 
     def perform(method, path, body = nil, headers = {})
-      headers = headers_for_user(@user).merge(headers)
-      Raven.capture_message('evss_headers', level: :info, extra: { evss_headers: headers })
-      super(method, path, body, headers)
+      merged_headers = @headers.merge(headers)
+      Raven.capture_message('evss_headers', level: :info, extra: { evss_headers: merged_headers })
```

Comment:
we originally added this to debug the gender issue; do we still need it?

Reply:
👍good call, I think we can take this out now